analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Sonar/LocksShouldBeReleased.Monitor.cs

```diff
@@ -0,0 +1,208 @@
+﻿using System;
+using System.Threading;
+
+namespace Tests.Diagnostics
+{
+    class Program
+    {
+        private object obj = new object();
+
+        public void DoSomethingWithMonitor1(bool b)
+        {
+            Monitor.Enter(obj); // FN
+            if (b)
```

Comment:
I slightly prefer 
```
if (condition)
```

Reply:
_Hint:_ `condition` could be a class field to remove the noise from method arguments